content/applications/inventory_and_mrp/purchase/advanced/vendor_costs_report.rst

```diff
@@ -82,8 +82,9 @@ measures to remove them, if desired.
 View results
 ============
 
-The vendor costs report is displayed in the *pivot* view, by default. Click :guilabel:`Insert in
-Spreadsheet` to add the pivot view into an editable spreadsheet format within the *Documents* app.
+After all of the filters and measures have been selected, the report generates in the pivot view.
+Click :guilabel:`Insert in Spreadsheet` to add the pivot view into an editable spreadsheet format
+within the *Documents* app.
```

Comment:
Since this is in a heading level that is not below the _Add filters and groups_ heading, where the filters and measures are selected, we can't assume the user is already in pivot view– they may be in the default graph view.

I suggest adding an internal reference to the section so that you can link directly to the filters and measures that are being referenced :)

Also add the custom anchor above the "Add filters and groups" heading
```
.. _purchase/vender-cost-report-filters:
```
```suggestion
After all of the :ref:`filters and measures have been selected
<purchase/vender-cost-report-filters>`, the report generates in the pivot view. Click
:guilabel:`Insert in Spreadsheet` to add the pivot view into an editable spreadsheet format within
the *Documents* app.
```